pyaerocom/aeroval/experiment_output.py

```diff
@@ -674,6 +674,9 @@ def _check_ovar_mvar_entry(self, mcfg, mod_var, ocfg, obs_var):
             if obs_var in mrv and mrv[obs_var] == mod_var:
                 # obs variable is in model_rename_vars
                 return True
+            elif mod_var in get_aliases(obs_var):
```

Comment:
not sure if I understand https://github.com/metno/pyaerocom/blob/ac9978576c35316ee4224164e7ae1508bed475e8/pyaerocom/aeroval/experiment_output.py#L632-L658 entirely. It might be that L632 should be 
```
mvar_aliases = get_aliases(obs_var)
```

Reply:
because https://github.com/metno/pyaerocom/blob/ac9978576c35316ee4224164e7ae1508bed475e8/pyaerocom/data/aliases.ini
does not define aliases for the sconc* variables (but only sconc* being an alias for conc*) 
https://github.com/metno/pyaerocom/blob/ac9978576c35316ee4224164e7ae1508bed475e8/pyaerocom/aeroval/experiment_output.py#L632 does not make any sense at all. This really should be
```
mvar_aliases = get_aliases(obs_var)
```

Reply:
due to this comment
https://github.com/metno/pyaerocom/blob/1538aa14d850c7862fe35c7ecbe4411a4f53218c/pyaerocom/aeroval/experiment_output.py#L666-L669

I will not change https://github.com/metno/pyaerocom/blob/1538aa14d850c7862fe35c7ecbe4411a4f53218c/pyaerocom/aeroval/experiment_output.py#L632 for now

The automatic use case and the CAMS61 setup works according to @hansbrenna . Converting the draft PR to a PR therefore